From 2fec8b8df6feac46d2aa1c1d5bd5eadc5d9fd887 Mon Sep 17 00:00:00 2001
From: Julian Bouzas <julian.bouzas@collabora.com>
Date: Thu, 21 Aug 2025 18:04:39 -0400
Subject: [PATCH 4/5] apply-routes.lua: Always set save=false when applying
 routes

When applying new routes, we always want the save property to be false because
it is done by WirePlumber (not the user). WirePlumber applies routes when
restoring them from the state file, or when finding the best routes after the
profile has changed. In both cases, it does not make sense to set save=true.
---
 src/scripts/device/apply-routes.lua | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/scripts/device/apply-routes.lua b/src/scripts/device/apply-routes.lua
index ce71a564..f74c337f 100644
--- a/src/scripts/device/apply-routes.lua
+++ b/src/scripts/device/apply-routes.lua
@@ -83,7 +83,7 @@ AsyncEventHook {
             index = route_info.index,
             device = device_id,
             props = Pod.Object (props),
-            save = route_info.save,
+            save = false,
           }
 
           log:debug (param,
-- 
2.50.1

